tests/e2e-legacy/preview.mjs

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+import { execSync } from "child_process";
+import { getDirName } from "./getDirName.mjs";
+import { getAllTags } from "./getAllTags.mjs";
+import { getPackageTags } from "./getPackageTags.mjs";
+import { runTestForTags } from "./runTestForTags.mjs";
+import conventionalCommitsParser from "conventional-commits-parser";
```

Comment:
make this a direct devDependency?

Reply:
Since we just needed `scope`, I used string APIs in [`06dacde` (#6538)](https://github.com/aws/aws-sdk-js-v3/pull/6538/commits/06dacde53054e8367bcdf8d972de8f8ac56fdc65)